src/main/java/com/direwolf20/buildinggadgets/common/schema/BlockData.java

```diff
@@ -0,0 +1,59 @@
+package com.direwolf20.buildinggadgets.common.schema;
+
+import com.google.common.base.MoreObjects;
+import net.minecraft.block.BlockState;
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.util.math.BlockPos;
+
+import javax.annotation.Nullable;
+import java.util.Objects;
+
+public final class BlockData {
```

Comment:
TemplateItem or TemplateData or TemplateBit to be consistent with the undo.

Reply:
As TemplateItem is kind of misleading and I don't like the Bit names, I'm gonna for TemplateData.